lib/video/screenRecording/firefox/firefoxWindowRecorder.js

```diff
@@ -42,9 +42,27 @@ function findRecordingDirectory(baseDir) {
   });
 
   log.debug('Using window recording directory: ' + directory);
+  if (directory === undefined) {
+    log.error('Could not find window recording directory in ' + baseDir);
+  }
   return directory;
 }
 
+function findRecordingStartTime(recordingDir) {
+  // Recording directory has the following format:
+  // .../browsertime-results/<web address>/<date time>/windowrecording-<unix recording start time>
+  return recordingDir.substr(recordingDir.lastIndexOf('-') + 1);
+}
+
+function findTimeToFirstFrame(recordingDir) {
+  const files = fs.readdirSync(recordingDir);
+  if (files === undefined || files.length < 1) {
+    log.error('Could not find recordings in ' + recordingDir);
+  } else {
+    return files[0].split('-')[2].split('.')[0];
```

Comment:
This still doesn't make sense to me.  Are you guaranteed to only have one file in this directory?